sponsor-dapp/src/components/ContractFinancialsTable.js

```diff
@@ -147,11 +153,11 @@ class ContractFinancialsTable extends Component {
 
             <TableRow key="remarginingFees">
               <TableCell>
-                <p>- Oracle Fees on remargin: </p>
+                <p>- Remargin fees: </p>
```

Comment:
Did you mean to include these cosmetic changes?

Reply:
Yeah there was a request to change the name.